### DISCUSSION TITLE
AFBR Carrier board connections

### DISCUSSION THREAD

**Discussion by rifi07** [OpenGammaProject/Open-Gamma-Detector#29] [Q&A] [answered]
Hi,

We have printed an AFBR carrier board, and we have soldered the AFBR sensor to it, but we don't understand how we have to connect it to the main board. I guess we have to connect the anode to the SiPM port (data output) and the cathode to the PSU to power it, but what about the GND? We have searched for the information, but we can not find it. 

Thanks

**Comment by NuclearPhoenixx** [comment, marked as answer]
It essentially comes down to if you wanna use the additional filtering on the board.

Like I described in the README, if you want to use that, you should solder the GND and the VCC connection, as well as closing the jumper. If you don't want to use it, you have to only solder the Cathode pad. Either way, the SiPM signal output is the Anode pad `A`, which will be connected to the `SIG` pin on the board.

`GND` on the carrier board is `GND` on the OGD (obviously). `VCC` or `C` on the carrier board are `PWR` on the device.

Hope this clears it up for you.

**Reply by rifi07** [reply]
In the case I don't want to use the filtering, I have to solder the anode pad to the SIG pad in the main board, the cathode to the PSU and GND to GND, right?

**Reply by NuclearPhoenixx** [reply]
yup!

**Reply by rifi07** [reply]
Ok, thank you very much!!

**Reply by NuclearPhoenixx** [reply]
Oh, wait. I forgot, you don't even need to solder GND without the filtering ;)
